src/pybind/mgr/zabbix/README.rst

```diff
@@ -0,0 +1,101 @@
+======================
+ceph-mgr zabbix module
+======================
+
+This module for ceph-mgr will send various items and statistics of a Ceph
+cluster to a Zabbix Server by invoking zabbix_sender.
+
+These items can be consumed inside Zabbix using 'Zabbix Trapper' items.
+
+Items being send to Zabbix include:
+- Overall Ceph status
+- I/O and bandwidth information
+- Number of OSDs (Total, UP, IN)
+- Storage utilization
+
+A XML template which works with this module can be found in the source directory
+of this module and should be easy to install and get up and running.
+
+Dependencies
+============
+
+On Ubuntu Xenial:
+
+::
+
+    apt install zabbix-agent
+
+On Fedora:
+
+::
+
+    dnf install zabbix-sender
+
+
+If you had already enabled the module, restart ceph-mgr after installing dependencies to reload the module.
+
+Enabling
+========
+
+Add this to your ceph.conf on nodes where you run ceph-mgr:
+
+::
+
+    [mgr]
+        mgr modules = zabbix
+
+If you use any other ceph-mgr modules, make sure they're in the list too.
+
+Restart the ceph-mgr daemon after modifying the setting to load the module.
+
+Configuring
+===========
+
+Two configuration keys are mandatory for the module to work:
+
+- mgr/zabbix/zabbix_host
+- mgr/zabbix/hostname
+
+The parameter *zabbix_host* controls the hostname of the Zabbix server to which
+*zabbix_sender* will send the items. This can be a IP-Address if required by
+your installation.
+
+The *hostname* parameter controls the hostname to use as source when sending items
+to Zabbix. This should match the name of the *Host* in your Zabbix server.
+
+Additional configuration keys which can be configured and their default values:
+
+- mgr/zabbix/zabbix_port: 10051
+- mgr/zabbix/zabbix_sender: /usr/bin/zabbix_sender
+- mgr/zabbix/interval: 60
+
+Configurations keys
+-------------------
+
+Configuration keys can be set on any machine with the proper cephx credentials,
+these are usually Monitors where the *client.admin* key is present.
+
+::
+
+    ceph config-key put <key> <value>
+
+For example:
+
+::
+
+    ceph config-key put mgr/zabbix/zabbix_host zabbix.localdomain
+    ceph config-key put mgr/zabbix/hostname ceph.eu-ams02.local
```

Comment:
It should be the same no matter what mgr sends it. It's the *identifier* of the *Host* configured in Zabbix.

Let's say you call the host *ceph.ams02* in Zabbix, it wants the key/value pairs to be send with that identifier.